lib/Caxy/HtmlDiff/HtmlDiff.php

```diff
@@ -131,12 +155,16 @@ private function processDeleteOperation($operation, $cssClass)
     {
         $text = array();
         foreach ($this->oldWords as $pos => $s) {
+            $matchFound = false;
             if ($pos >= $operation->startInOld && $pos < $operation->endInOld) {
-                if ($s === '[[REPLACE_TABLE]]' && isset($this->oldTables[$pos])) {
-                    foreach ($this->oldTables[$pos] as $word) {
+                foreach ($this->specialElements as $specialElement) 
```

Comment:
See above comment for how this can be refactored using `in_array`.

Also, it looks like this is missing the opening brace (`{`)?